spec.html

```diff
@@ -7939,7 +7939,8 @@ <h1>Objectives</h1>
     <emu-clause id="sec-liveness">
       <h1>Liveness</h1>
 
-      <p>For some set of objects _S_, a <dfn>hypothetical WeakRef-oblivious</dfn> execution with respect to _S_ is an execution whereby %WeakRef.prototype.deref% being called on a WeakRef whose referent is an element of _S_ always returns *undefined*.</p>
+      <p>For some set of objects _S_, a <dfn>hypothetical WeakRef-oblivious</dfn> execution with respect to _S_ is an execution whereby the abstract operation WeakRefDeref being called on a WeakRef whose referent is an element of _S_ always returns *undefined*.</p>
```

Comment:
Nit: I'm not sure if we say "being called on" for abstract operations; maybe this would be better expressed as "of"